compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1647,6 +1647,25 @@ static bool inheritsFromError(Type* t) {
 }
 
 
+static bool inheritsFromAbsBaseArr(Type* t) {
```

Comment:
Isn't this the same as `isArrayImplType` ?

Reply:
Good catch, thanks!